server.go

```diff
@@ -92,10 +92,12 @@ type Server struct {
 	databases map[string]*database // databases by name
 	shards    map[uint64]*Shard    // shards by id
 	users     map[string]*User     // user by name
+
+	version string // current version of the server
 }
 
 // NewServer returns a new instance of Server.
-func NewServer() *Server {
+func NewServer(version string) *Server {
```

Comment:
Hmmm. Something about this looks weird. I know why you are doing it, but it hurts my eyes. Is there another option?